django/db/migrations/operations/models.py

```diff
@@ -42,8 +42,38 @@ def __init__(self, name, fields, options=None, bases=None, managers=None):
         self.options = options or {}
         self.bases = bases or (models.Model,)
         self.managers = managers or []
+
         super(CreateModel, self).__init__(name)
 
+        # Sanity-check that there are no duplicated field names, bases, or
+        # manager names
+        used_field_names = set()
+        for name, _ in fields:
+            if name in used_field_names:
+                raise ValueError(
+                    'You cannot have more than one field with the same name. '
+                    '"%s" was used more than once' % name
+                )
+            used_field_names.add(name)
+
+        used_bases = set()
+        for base in tuple(self.bases):
```

Comment:
we still do - tuple requires an iterable, though you're right, there's no reason to use `tuple()` anymore